#!/usr/bin/python3


def raise_exception_msg(message=""):
    """Prints the provided message."""
    print(message)